src/utils/find-version-added.ts

```diff
@@ -0,0 +1,50 @@
+import { GitProcess } from 'dugite';
+import * as semver from 'semver';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import { chain } from 'lodash';
+
+const gitDir = '/Users/codebytere/Developer/electron-gn/src/electron';
```

Comment:
these paths are terribly wrong; i just needed them to work with wonkiness of `npm link`